src/plots/mapbox/layout_attributes.js

```diff
@@ -95,6 +95,35 @@ var attrs = module.exports = overrideAll({
         ].join(' ')
     },
 
+    westbound: {
```

Comment:
This is already a word with a different meaning - like "take the westbound train"

Perhaps `bounds: {north, south, east, west}`?

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/6339/commits/db225cde06f5977ebbd7f298e1f36eea620b5eb1.